editor/editor_build_profile.cpp

```diff
@@ -192,13 +416,24 @@ String EditorBuildProfile::get_build_option_description(BuildOption p_build_opti
 
 	const char *build_option_descriptions[BUILD_OPTION_MAX] = {
 		TTRC("3D Nodes as well as RenderingServer access to 3D features."),
-		TTRC("2D Physics nodes and PhysicsServer2D."),
-		TTRC("3D Physics nodes and PhysicsServer3D."),
-		TTRC("Navigation, both 2D and 3D."),
+		TTRC("Navigation Server and capabilities for 2D."),
+		TTRC("Navigation Server and capabilities for 3D."),
 		TTRC("XR (AR and VR)."),
+		TTRC("OpenXR standard implementation (requires XR to be enabled)."),
+		TTRC("Wayland display (Linux only)."),
+		TTRC("X11 display (Linux only)."),
 		TTRC("RenderingDevice based rendering (if disabled, the OpenGL back-end is required)."),
-		TTRC("OpenGL back-end (if disabled, the RenderingDevice back-end is required)."),
+		TTRC("Forward+ renderer for advanced 3D graphics."),
+		TTRC("Mobile renderer for less advanced 3D graphics."),
 		TTRC("Vulkan back-end of RenderingDevice."),
+		TTRC("Direct3D 12 back-end of RenderingDevice."),
```

Comment:
This is the only file that uses "back-end" with a hyphen. I'd suggest streamlining it to "backend" which seems more common, since we're changing those strings.